Remove the default features from wgpu-info (#2753)
gfx-rs/wgpu: wgpu-info/README.md

#### Listing Adapters

When called with no arguments, wgpu-info will list all adapters visible to wgpu and all the information about them we have.

> For gl in non Linux add the angle feature, for vk on macOS add the vulkan-portability feature.

Reviewer: ```suggestion

For gl in non Linux add the angle feature, for vk on macOS add the vulkan-portability feature.
```

When called with no arguments, wgpu-info will list all adapters visible to wgpu and all the information about them we have.

> For gl in non Linux add the angle feature, for vk on macOS add the vulkan-portability feature.

Reviewer: ```suggestion
For OpenGL on platforms other than Linux add the `angle` feature, for Vulkan on macOS add the `vulkan-portability` feature.
```
